scripts/scripts.js

```diff
@@ -354,6 +354,39 @@ function decorateH3(main) {
     });
   }
 }
+
+/**
+ * Checks for the same domain or not and if ends with pdf
+ */
+function isSameDomainOrPdf(url) {
+  const ancUrl = (url.indexOf('://') > 0 || url.indexOf('//') === 0)
+    ? new URL(url)
+    : new URL(window.location.origin + url);
+  return (
+    window.location.hostname === ancUrl.hostname
+    || ancUrl.pathname.toLowerCase().endsWith('.pdf')
+    || ancUrl.pathname.toLowerCase().endsWith('.hlx.live')
+    || ancUrl.pathname.toLowerCase().endsWith('.hlx.page')
```

Comment:
These checks for `.hlx.live` and `.hlx.page` should be on the url domain/hostname, not on the pathname.